### PR TITLE
Centralise the Caption , Variable Size Post Shifting Fixing , Name fo…

### PR DESCRIPTION
**Pull request by Gurmeet-Singh657**
✅ Name not repeating in caption
image

✅ Centralised the Caption
image

✅ Post having captions now has fixed size and can be expanded using Read More and Read Less
image

### CODE REVIEW

**Comment by Gurmeet-Singh657** [comment]
@narayan954 Please merge the pull request with the given changes so that I can work upon other issue !!

**Comment by narayan954** [comment]
> @narayan954 Please merge the pull request with the given changes so that I can work upon other issue !!

looking into it asap

**Comment by narayan954** [comment]
> @narayan954 Please merge the pull request with the given changes so that I can work upon other issue !!

mention the issue number, I'm telling you again

File: src/components/Post.jsx [hunk 1]
```diff
@@ -201,6 +201,11 @@ function Post(prop) {
     setisCommentOpen(false);
   };
 
+  const [readMore, setReadMore] = useState(false);
```
Comment (narayan954) [inline src/components/Post.jsx]: useState variables should not be defined here

**Review by narayan954 — APPROVED** [review]

File: src/components/Post.jsx (hunk 1 above)
Comment (narayan954) [reply src/components/Post.jsx]: i relocated them

**Comment by narayan954** [comment]
Thanks for the contribution, but please mention issue number and try formatting the code before committing. You may try prettier or any formatter

**Pull request merged**